context-free.md

```diff
@@ -1,22 +1,30 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+It is for image designers or graphical artists. 
 
 ## What is easy to do in this language? Why is it easy?
-
+It is really easy to make squares, circles, and triangles of various colors, shapes, sizes, and locations.
```

Comment:
This is fairly common in a lot of low-level art programs - you can build pretty much whatever you want if you have paths and simple shapes.